assets/app/view/game/part/blocker.rb

```diff
@@ -54,7 +54,7 @@ class Blocker < Base
         }.freeze
 
         def preferred_render_locations
-          if @tile.parts.reject(&:border?).one?
+          if @tile.parts.reject(&:border?).empty?
```

Comment:
tile.parts.none?(:border?)

Reply:
I don't think that is the same function.

Existing: Are there any parts besides borders?
Yours: Are there no borders?

Reply:
Perhaps !@tile.parts.any? { |p| !p.border? }

Reply:
OK

Reply:
But it should be @tile.parts.none? { |p| !p.border? }

Reply:
@tile.all?(&:border?)